### PR TITLE
Fixed the 2 environment issues bash head and eth0 interface

### PR DESCRIPTION
**Pull request by jshaofuturewei**
Fixed 2 environment issues
1. Commented /usr/bin/env bash 
2.  Add check condition statement for eth0

### CODE REVIEW

File: hack/arktos-up.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/s!/usr/bin/env bash
+#/s!/usr/bin/env bash
```
Comment (pdgetrf) [inline hack/arktos-up.sh:1]: did you mean #!/usr/bin/env bash

Comment (jshaofuturewei) [reply hack/arktos-up.sh]: Yes.

File: hack/lib/etcd.sh [hunk 2]
```diff
@@ -18,9 +18,14 @@
 # A set of helpers for starting/running etcd for tests
 
 ETCD_VERSION=${ETCD_VERSION:-3.3.10}
-INT_NAME="eth0"
+INT_NAME=${INT_NAME:-eth0}
```
Comment (vinaykul) [inline hack/lib/etcd.sh:21]: This will still default to eth0 if not specified. Please add code to query the default interface using ip route.

Comment (jshaofuturewei) [reply hack/lib/etcd.sh]: Updated it to ip route | awk '/default/ { print $5 }'

**Pull request merged**